src/python/python-proto/python_proto/api/graph_query_proxy/v1beta1/messages.py

```diff
@@ -0,0 +1,96 @@
+from __future__ import annotations
```

Comment:
## These 4 classes are exactly the same as the ones in `graph_query/messages.py` except for the removal of `tenant_id` in the requests.